재호/1935.py

```diff
@@ -0,0 +1,37 @@
+def is_expression(char):
+    return char == '*' or char == '+' or char == '/' or char == '-'
+
+
+def cal(a, b, exp):
```

Comment:
중요하지 않은 부분 일 수 있는데 cal라는 네이밍은 이름만 보고 의도가 뭔지 파악하기 힘들 수 있을 것 같습니다. 
```suggestion
def calculate(a, b, exp):
```